ngc fails when using Telerik platform's everlive-sdk

The ngc compilation fails when using the everlive-sdk plugin (see this [forum post](http://www.telerik.com/forums/aot-not-working#hPf9ChuuSkONMPPrw1Wy2Q)):

```
ERROR in ../~/node-localstorage/LocalStorage.js
Module not found: Error: Can't resolve 'fs' in '/Users/joni/projects/testjoni/node_modules/node-localstorage'
 @ ../~/node-localstorage/LocalStorage.js 9:7-20
 @ ../~/everlive-sdk/dist/everlive.all.js
 @ ./providers/backend.ts
 @ ./app.module.ngfactory.ts
 @ ./main.aot.ts
```

According to Georgi who helped me out there the `webpack.common.js` should disable the fs shim that is conflicting with NS:

```
node: {
        //Disable node shims that conflict with NativeScript
        "http": false,
        "timers": false,
        "setImmediate": false,
        "fs": "empty",
    }
```

--- a/webpack.common.js.angular.template
+++ b/webpack.common.js.angular.template
@@ -104,6 +104,7 @@ module.exports = function (platform, destinationApp) {
             "http": false,
             "timers": false,
             "setImmediate": false,
+            "fs": "empty",
         },
         module: {
             loaders: [

--- a/webpack.common.js.nativescript.template
+++ b/webpack.common.js.nativescript.template
@@ -90,6 +90,7 @@ module.exports = function (platform, destinationApp) {
             "http": false,
             "timers": false,
             "setImmediate": false,
+            "fs": "empty",
         },
         module: {
             loaders: [

